app/assets/stylesheets/invitations.css.scss

```diff
@@ -5,11 +5,19 @@
 }
 
 #invitationsModal {
+  #paste_link_wrapper {
```

Comment:
Not a best practice, but in this case, I think it does a good job preventing any style collisions or cascading mistakes...

Anyway, using camelCase and under_scores mixed on two lines of code is not recommended either... :(